src/System.Private.Xml/src/System/Xml/Serialization/Configuration/SchemaImporterExtensionsSection.cs

```diff
@@ -11,77 +11,12 @@ namespace System.Xml.Serialization.Configuration
-            this.SchemaImporterExtensions.Add(
-                new SchemaImporterExtensionElement(ConfigurationStrings.SqlTypesSchemaImporterBigInt, GetSqlTypeSchemaImporter("TypeBigIntSchemaImporterExtension")));
-            this.SchemaImporterExtensions.Add(
-                new SchemaImporterExtensionElement(ConfigurationStrings.SqlTypesSchemaImporterInt, GetSqlTypeSchemaImporter("TypeIntSchemaImporterExtension")));
-            this.SchemaImporterExtensions.Add(
-                new SchemaImporterExtensionElement(ConfigurationStrings.SqlTypesSchemaImporterSmallInt, GetSqlTypeSchemaImporter("TypeSmallIntSchemaImporterExtension")));
-            this.SchemaImporterExtensions.Add(
-                new SchemaImporterExtensionElement(ConfigurationStrings.SqlTypesSchemaImporterTinyInt, GetSqlTypeSchemaImporter("TypeTinyIntSchemaImporterExtension")));
-            this.SchemaImporterExtensions.Add(
-                new SchemaImporterExtensionElement(ConfigurationStrings.SqlTypesSchemaImporterBit, GetSqlTypeSchemaImporter("TypeBitSchemaImporterExtension")));
-            this.SchemaImporterExtensions.Add(
-                new SchemaImporterExtensionElement(ConfigurationStrings.SqlTypesSchemaImporterFloat, GetSqlTypeSchemaImporter("TypeFloatSchemaImporterExtension")));
-            this.SchemaImporterExtensions.Add(
-                new SchemaImporterExtensionElement(ConfigurationStrings.SqlTypesSchemaImporterReal, GetSqlTypeSchemaImporter("TypeRealSchemaImporterExtension")));
-            this.SchemaImporterExtensions.Add(
-                new SchemaImporterExtensionElement(ConfigurationStrings.SqlTypesSchemaImporterDateTime, GetSqlTypeSchemaImporter("TypeDateTimeSchemaImporterExtension")));
-            this.SchemaImporterExtensions.Add(
-                new SchemaImporterExtensionElement(ConfigurationStrings.SqlTypesSchemaImporterSmallDateTime, GetSqlTypeSchemaImporter("TypeSmallDateTimeSchemaImporterExtension")));
-            this.SchemaImporterExtensions.Add(
-                new SchemaImporterExtensionElement(ConfigurationStrings.SqlTypesSchemaImporterMoney, GetSqlTypeSchemaImporter("TypeMoneySchemaImporterExtension")));
-            this.SchemaImporterExtensions.Add(
-                new SchemaImporterExtensionElement(ConfigurationStrings.SqlTypesSchemaImporterSmallMoney, GetSqlTypeSchemaImporter("TypeSmallMoneySchemaImporterExtension")));
-            this.SchemaImporterExtensions.Add(
-                new SchemaImporterExtensionElement(ConfigurationStrings.SqlTypesSchemaImporterUniqueIdentifier, GetSqlTypeSchemaImporter("TypeUniqueIdentifierSchemaImporterExtension")));
-        }
-
-        protected override ConfigurationPropertyCollection Properties
-        {
-            get { return _properties; }
-        }
-
-        public SchemaImporterExtensionElementCollection SchemaImporterExtensions
-        {
-            get { return null; }
-        }
-
         internal SchemaImporterExtensionCollection SchemaImporterExtensionsInternal
```

Comment:
Looks like even this last member is dead so we should just delete this entire type.